script/brave_license_helper.py

```diff
@@ -141,7 +141,9 @@ def AddBraveCredits(prune_paths, special_cases, prune_dirs, additional_paths):
 
 def CheckBraveMissingLicense(target_os, path, error):
     if path.startswith('brave'):
-        if (target_os == 'android'):
+        if (target_os == 'ios'):
+            return # iOS does not add licenses
```

Comment:
What build failures are you seeing that require this?

Reply:
I get build failures locally only when I dont apply this change, I have no idea why 🙁

```
[10223/18658] ACTION //components/resources:about_credits(//build/toolchain/mac:ios_clang_x64)
FAILED: gen/components/resources/about_credits.html 
python ../../tools/licenses.py --target-os=ios --depfile gen/components/resources/about_credits.d credits gen/components/resources/about_credits.html
Traceback (most recent call last):
  File "../../tools/licenses.py", line 832, in <module>
    sys.exit(main())
  File "../../tools/licenses.py", line 817, in main
    args.gn_target, args.depfile):
  File "../../tools/licenses.py", line 687, in GenerateCredits
    CheckBraveMissingLicense(target_os, path, e)
  File "/Users/kyle/git/brave/brave-browser/src/brave/script/brave_license_helper.py", line 150, in CheckBraveMissingLicense
    raise error
__main__.LicenseError: missing README.chromium or licenses.py SPECIAL_CASES entry in brave/out/Debug/obj/third_party/sqlite
```

Reply:
Maybe this `brave/out/Debug` folder shouldn't be there?

Reply:
Yeah that's odd. When I build for desktop, the `out` directory is in `src/out/`, not `src/brave/out/`.

Does it get re-created if you delete? I'm not familiar with how the iOS browser gets built.

Reply:
If it is an expected directory, then we should add it to https://github.com/brave/brave-core/blob/0ad97cd66022ca861cd2a61ba8525083885f0ca1/script/brave_license_helper.py#L123 to fix the build error you're seeing.

Reply:
@fmarier When I tried to delete the folder I got other build errors which is more odd… Perhaps I will try to do a straight up clean build and see what happens. It could be an misconfigured iOS build config of some sort